docs/source/pipelines.rst

```diff
@@ -4,194 +4,321 @@
-            ratios for the |split step| of the :ref:`MLflow Regression Pipeline
-            <mlflow-regression-pipeline>`.  This configuration demonstrates use of
-            |Jinja2| templating syntax to define a dynamic value for the ``split_ratios``
-            attribute that can be overriden by
-            :ref:`Pipeline Profiles <profiles-key-concept>`.
-
-  steps:
-    split:
-      # Train/validation/test split ratios
-      split_ratios: {{SPLIT_RATIOS|default([0.75, 0.125, 0.125])}}
+- **Templates**: A Pipeline Template is a git repository with a :ref:`standardized, modular layout
+  <pipeline-template-structure>` containing all of the customizable code and configurations for a
+  :ref:`Pipeline <pipelines-key-concept>`. Each template also defines its requirements, data science
+  notebooks, and tests. MLflow Pipelines includes :ref:`predefined templates <pipeline-templates>`
+  for a variety of model development and MLOps tasks.
 
-.. _pipelines-key-concept:
+.. _profiles-key-concept:
 
-Pipelines
-~~~~~~~~~
+- **Profiles**: Profiles contain user-specific or environment-specific configurations for a
+  :ref:`Pipeline <pipelines-key-concept>`, such as the particular set of hyperparameters being
+  tuned by a data scientist in development or the MLflow Model Registry URI and credentials
+  used to store production-worthy models. Each profile is represented as a YAML file in the
+  :ref:`Pipeline Template <pipeline-templates-key-concept>`.
 
-A Pipeline is a composition of :ref:`Steps <steps-key-concept>` used to solve an ML problem
-or perform an MLOps task, such as developing an accurate regression model for data within a
-particular domain or distributing a model to perform scalable batch inference. Pipelines
-structure their code, configurations, and tests as git repositories with a standardized
-directory hierarchy, as discussed in :ref:`Pipeline Repositories
-<pipeline-repositories-key-concept>`. MLflow Pipelines includes predefined pipeline repositories
-for common ML problems and MLOps tasks called :ref:`Pipeline Templates <pipeline-templates>`.
-MLflow Pipelines also provides :py:class:`APIs <mlflow.pipelines.Pipeline>` and a :ref:`CLI <cli>`
-for running pipelines and inspecting their results.
+.. _step-cards-key-concept:
 
-.. code-section::
+- **Step Cards**: Step Cards display :ref:`Pipeline <pipelines-key-concept>` results, including
+  dataset profiles, model performance & explainability plots, overviews of the best model
+  parameters found during tuning, and more. Step Cards and their corresponding dataset and model
+  information are also logged to MLflow Tracking.
 
-    .. code-block:: python
-      :caption: Example API and CLI workflows for running the :ref:`MLflow Regression Pipeline
-                <mlflow-regression-pipeline>` and inspecting results. Note that pipelines
-                must be run from within their corresponding git repositories.
+Usage
```

Comment:
@jinzhang21 This section is unchanged, as usage updates are assigned to you via ML-24619. Feel free to push changes to this PR or file a separate PR.